### ISSUE TITLE
纯html 引入报错  Error: Can only apply to VectorLayer or VectorTileLayer，是不支持吗

### ISSUE THREAD

**Issue by WuXinZhiMing** [openlayers/ol-mapbox-style#446]
代码页面  http://82.156.12.5/ol/openlayerpbf.html
纯html引入olms，报错 index.js:168 Uncaught (in promise) Error: Can only apply to VectorLayer or VectorTileLayer
![Snipaste_2022-01-26_15-25-22](https://user-images.githubusercontent.com/31824814/151120741-2f84770f-7556-48c7-bc94-377f22a40d7d.png)
![Snipaste_2022-01-26_15-26-19](https://user-images.githubusercontent.com/31824814/151120751-47e13b07-39cc-4181-8e60-0f033482af9a.png)

```
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>openlayers 自定义地图样式</title>
    <style>
        #map {
            width: 100%;
            height:800px;
        }
    </style>
    <link rel="stylesheet" href="http://82.156.12.5/ol/6.9js/ol.css">
    <script src="http://82.156.12.5/ol/6.9js/ol.js"></script>
     <script src="http://82.156.12.5/ol/ol-mapbox-style/dist/olms.js"></script>
</head>
<body>
<div id="map" class="map"></div>
</body>
<script>
       var gridsetName = 'EPSG:4326';
       var gridNames = ['EPSG:4326:0', 'EPSG:4326:1', 'EPSG:4326:2', 'EPSG:4326:3', 'EPSG:4326:4', 'EPSG:4326:5', 'EPSG:4326:6', 'EPSG:4326:7', 'EPSG:4326:8', 'EPSG:4326:9', 'EPSG:4326:10', 'EPSG:4326:11', 'EPSG:4326:12', 'EPSG:4326:13', 'EPSG:4326:14', 'EPSG:4326:15', 'EPSG:4326:16', 'EPSG:4326:17', 'EPSG:4326:18', 'EPSG:4326:19', 'EPSG:4326:20', 'EPSG:4326:21'];
       var baseUrl = 'http://82.156.12.5:9006/geoserver/gwc/service/wmts';
       var style = '';
       var format = 'application/vnd.mapbox-vector-tile';
       var infoFormat = 'text/html';
       var layerName = 'map:mapcq';
       var projection = new ol.proj.Projection({
           code: 'EPSG:4326',
           units: 'degrees',
           axisOrientation: 'neu'
       });
       var resolutions = [0.703125, 0.3515625, 0.17578125, 0.087890625, 0.0439453125, 0.02197265625, 0.010986328125, 0.0054931640625, 0.00274658203125, 0.001373291015625, 6.866455078125E-4, 3.4332275390625E-4, 1.71661376953125E-4, 8.58306884765625E-5, 4.291534423828125E-5, 2.1457672119140625E-5, 1.0728836059570312E-5, 5.364418029785156E-6, 2.682209014892578E-6, 1.341104507446289E-6, 6.705522537231445E-7, 3.3527612686157227E-7];
       var params = {
           'REQUEST': 'GetTile',
           'SERVICE': 'WMTS',
           'VERSION': '1.0.0',
           'LAYER': layerName,
           'STYLE': style,
           'TILEMATRIX': gridsetName + ':{z}',
           'TILEMATRIXSET': gridsetName,
           'FORMAT': format,
           'TILECOL': '{x}',
           'TILEROW': '{y}'
       };

       var url = baseUrl+'?'
       for (var param in params) {
           url = url + param + '=' + params[param] + '&';
       }
       url = url.slice(0, -1);

    layer = new ol.layer.VectorTile({
       source: new ol.source.VectorTile({
           url: url,
           format: new ol.format.MVT({}),
           projection: projection,
           tileGrid: new ol.tilegrid.WMTS({
               tileSize: [256,256],
               origin: [-180.0, 90.0],
               resolutions: resolutions,
               matrixIds: gridNames
           }),
           wrapX: true
       }) ,
   });


   var view = new ol.View({
       center: [0, 0],
       zoom: 2,
       resolutions: resolutions,
       projection: projection,
       extent: [-180.0,-90.0,180.0,90.0]
   });


   var map = new ol.Map({
       controls: ol.control.defaults({attribution: false}).extend([
           new ol.control.MousePosition(),
       ]),
       layers: [layer],
       target: 'map',
       view: view
   });



 fetch(
     'http://82.156.12.5/ol/ol-mapbox-style/root.json'
     )
     .then(res => res.json().then(json => {
         olms.applyStyle(layer, json,undefined , undefined, layer.getSource().getTileGrid().getResolutions());
     }));

</script>

</html>
```

**Comment by ahocevar** [comment]
It looks like the externals `ol/layer/Vector` and `ol/layer/VectorTile` from the OpenLayers build (`ol.js`) are not picked up by the ol-mapbox-style build (`olms.js`). Are these `.js` files official builds, or your own? If the latter, how did you build them?

**Comment by ahocevar** [comment]
This is a bug in the build configuration and was fixed with #501.
